hero/models.py

```diff
@@ -692,13 +698,18 @@ def sync_from_discord_obj(cls, discord_obj):
         return obj, not created
 
     async def fetch(self) -> discord.Message:
-        if not self.channel.is_fetched:
-            await self.channel.fetch()
-        discord_message = self.channel.get_message(self.id)
-        if discord_message is None:
-            discord_message = await self.channel.fetch_message(self.id)
+        channel_obj = await self.channel
+        if not channel_obj.is_fetched:
+            await channel_obj.fetch()
+        discord_message = await channel_obj.fetch_message(self.id)
         self._discord_obj = discord_message
         return discord_message
+    
+    def __str__(self):
+        if self.is_fetched:
+            return self.content
```

Comment:
While this might work it's probably better to use `self._discord_obj.content` instead.

Reply:
Done